src/style.css

```diff
@@ -1,3 +1,85 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--main-color);
+}
+
+body,
+.header,
+.header-logo,
+.menu,
+.menu-item,
+.menu-link,
+::after {
+  box-sizing: border-box;
+}
+
+.header {
+  width: 100vw;
```

Comment:
Unless there's a specific requirement to do so, you should avoid fixing container sizes. Let the content size dictate it. This is to avoid overflow or an accidental scroll bar. In this case, you've set the width of `.header` to `100vw`, which might lead to horizontal scrolling due to vertical scrollbars occupying some space.